### ISSUE TITLE
[BUG] <output_scores输出为什么存在-inf的值>

### ISSUE THREAD

**Issue by DJinsis** [QwenLM/Qwen-VL#192]
查看了qwen_generation_utols.py里面的代码，top_k_logits函数对logits进行了处理，所以会存在值为-inf的情况，但是我把这段代码注释掉了，直接return logits，得到的值还是有-inf，这种情况正常么

**Comment by ShuaiBai623** [comment]
是transformer库中的logitprocess里的修改导致
